apps/xomg_tasks/lib/utils.ex

```diff
@@ -20,6 +20,17 @@ defmodule XomgTasks.Utils do
   @doc """
   Runs a specific app for some arguments. Will handle IEx, if one's running
   """
+  def generic_run(args, apps) when is_list(apps) do
```

Comment:
maybe, for a slightly DRYer code at little cost, lets leave only the new clause of `generic_run` and use a single-element list if there's just one app to start?